src/main/kotlin/com/autonomousapps/tasks/FindDeclaredProcsTask.kt

```diff
@@ -55,6 +56,11 @@ abstract class FindDeclaredProcsTask : DefaultTask() {
   init {
     group = TASK_GROUP_DEP_INTERNAL
     description = "Produces a report of all supported annotation types and their annotation processors"
+
+    if (GradleVersions.isAtLeastGradle74) {
+      @Suppress("LeakingThis")
+      notCompatibleWithConfigurationCache("Cannot serialize unsafeLazy")
```

Comment:
Or could be renamed to `InMemoryCache`
```suggestion
      notCompatibleWithConfigurationCache("Cannot serialize InMemoryCache")
```